### PR TITLE
CIRCSTORE-177 Add effective Overdue Fine Policy and Lost Item Policy to loan schema

### PR DESCRIPTION
**Pull request by Svitlana-Zmiivska**
**Purpose**
Add effective Overdue Fine Policy and Lost Item Policy to loan schema

**Link**
[CIRCSTORE-177](https://issues.folio.org/browse/CIRCSTORE-177)

### CODE REVIEW

File: ramls/loan.json [hunk 1]
```diff
@@ -98,6 +98,14 @@
       "type" : "string",
       "format": "date-time"
     },
+    "effectiveOverdueFinePolicy": {
```
Comment (bohdan-suprun) [inline ramls/loan.json]: Is it an Id identifier?

Comment (Svitlana-Zmiivska) [reply ramls/loan.json]: Good point. It should be an identifier

File: ramls/loan.json [hunk 2]
```diff
@@ -98,6 +98,16 @@
       "type" : "string",
       "format": "date-time"
     },
+    "effectiveOverdueFinePolicyId": {
```
Comment (marcjohnson-kint) [inline ramls/loan.json]: What is the effective overdue fine policy? Why is that different to the lost item policy?

Comment (Svitlana-Zmiivska) [reply ramls/loan.json]: The effective overdue fine policy is needed to facilitate the quick determination of overdue fines at the time the item is check-in or renewed. 
The lost item policy is needed to facilitate the quick determination  of when the item ages to lost and the associated fees or the associated fees if the patron declares the item lost.

Comment (Svitlana-Zmiivska) [reply ramls/loan.json]: You can find more details about these fields in the requirements of the story https://issues.folio.org/browse/CIRC-591

Comment (marcjohnson-kint) [reply ramls/loan.json]: I don't think I'm explaining myself well.

These properties appear to be for remembering the policies in use for that loan. What I'm trying to understand is why one is prefixed with `effective` and the other isn't.

`effective` is typically used for when a property is derived from related properties in a record hierarchy (e.g. items and holdings, which is the only place I'm aware of that it has been used).

Comment (Svitlana-Zmiivska) [reply ramls/loan.json]: Changed

**Pull request merged**